installer/dev/main.cpp

```diff
@@ -31,6 +31,19 @@ int wmain(int argc, wchar_t *argv[])
         }
     }
 
-    return ProjectReunionInstaller::DeployPackages(quiet);
+    const HRESULT deployPackagesResult = ProjectReunionInstaller::DeployPackages(quiet);
+    if (!quiet)
+    {
+        if (SUCCEEDED(deployPackagesResult))
+        {
+            std::cout << "All packages were installed successfully." << std::endl;
+        }
+        else
+        {
+            std::wcerr << "One or more packages failed to install. Result: 0x" << std::hex << deployPackagesResult << std::endl;
```

Comment:
NIT: cout vs wcerr. Probably best to pick 1 and be consistent cout+cerr or (better) wcout+wcerr.

For a future cleanup PR